mono/mini/method-to-ir.c

```diff
@@ -3931,7 +3931,6 @@ mono_method_check_inlining (MonoCompile *cfg, MonoMethod *method)
 		inline_limit_inited = TRUE;
 	}
 
-#ifdef ENABLE_NETCORE
 	if (COMPILE_LLVM (cfg)) {
```

Comment:
This was added to avoid possible regressions in non-netcore cases.

Reply:
I suspected that was the case, so in practice it means that none netcore will get the AOT inline limit instead of the old AOT limit, but maybe we could set defaults to the same on none netcore build instead, meaning that you can do this define on the defaults instead, when the limit is actually set they all get the same value and don't think there is a way to set different values as part of env variable.

Reply:
So maybe something like this:

```
#ifdef ENABLE_NETCORE
/* Used by LLVM AOT */
#define LLVM_AOT_INLINE_LENGTH_LIMIT 30
#else
#define LLVM_AOT_INLINE_LENGTH_LIMIT INLINE_LENGTH_LIMIT
#endif
```

Reply:
Made above change, so with that we should get same default on none netcore build as before, meaning on the following default inline length limit:

mini JIT: 20
LLVM JIT: 100
all AOT (mini+LLVM): 20


On netcore we will get the following inline length limit (same as before):

mini JIT: 20
mini AOT: 20
LLVM JIT: 100
LLVM AOT: 30